src/Calypso-SystemPlugins-SUnit-Browser/ClyJumpToTestMethodCommand.class.st

```diff
@@ -0,0 +1,103 @@
+"
+Description
+--------------------
+
+I am a command available in the method context menu of Calypso. When activated I'll jump on the test method corresponding to the method. 
+
+If the method does not exists I will create it in the corresponding class.
+
+Internal Representation and Key Implementation Points.
+--------------------
+
+    Instance Variables
+	selectedMethods:			<aCollection>	Methods selected by the user that should have their tests generated.
+	testMethodToBrowse:		<aMethod>			The method to browse at the end of the execution of the command.
+
+"
+Class {
+	#name : #ClyJumpToTestMethodCommand,
+	#superclass : #SycMethodCommand,
+	#traits : 'TClyGenerateTestClass',
+	#classTraits : 'TClyGenerateTestClass classTrait',
+	#instVars : [
+		'testMethodToBrowse',
+		'browser',
+		'systemEnvironment'
+	],
+	#category : #'Calypso-SystemPlugins-SUnit-Browser-Commands'
+}
+
+{ #category : #testing }
+ClyJumpToTestMethodCommand class >> canBeExecutedInContext: aBrowserContext [
+	^ aBrowserContext isInstanceSideMethodSelected and: [ aBrowserContext selectedItems anySatisfy: [ :each | (each hasProperty: ClyTestResultProperty) not ] ]
+]
+
+{ #category : #activation }
+ClyJumpToTestMethodCommand class >> methodMenuActivation [
+	<classAnnotation>
+	^ CmdContextMenuActivation byItemOf: ClySUnitMethodMenuGroup order: 10 for: ClyMethod asCalypsoItemContext
+]
+
+{ #category : #activation }
+ClyJumpToTestMethodCommand class >> methodShortcutActivation [
+	<classAnnotation>
+	^ CmdShortcutActivation by: $h meta , $j meta for: ClyMethod asCalypsoItemContext
+]
+
+{ #category : #accessing }
+ClyJumpToTestMethodCommand >> defaultMenuItemName [
+	^ 'Jump to test method'
+]
+
+{ #category : #execution }
+ClyJumpToTestMethodCommand >> execute [
+	self methods ifEmpty: [ self inform: 'No method selected'. ^ self ].
```

Comment:
It not necessary because superclass defines condition that command will not be active when nothing is selected

Reply:
Updated.